src/main/java/frc/team5115/Commands/Auto/VisionAuto/DriveForwardWVision.java

```diff
@@ -12,12 +12,18 @@ public class DriveForwardWVision extends CommandBase{
     private double speed;
     private boolean doneMoving;
     private Timer timer;
-    private final double timeout;
+    private double timeout;
+    private double photonPoseEstimator;
 
-    public DriveForwardWVision(Drivetrain drivetrain, double dist, double speed) {
+    public DriveForwardWVision(Drivetrain drivetrain2, double d, double e) {
+    
+    }    
+
+    public void DriveForwardWVision(Drivetrain drivetrain, double dist, double speed, double photonPoseEstimator) {
         this.dist = dist;
         this.drivetrain = drivetrain;
         this.speed = speed;
+        photonPoseEstimator = photonPoseEstimator;
```

Comment:
There's a bug here. If you give an instance variable the same name as a constructor argument, you are assigning the local variable to itself.

The correct syntax here is
`this.photoPoseEstimator = photoPoseEstimator`